packages/oui-collapsible/src/collapsible.html

```diff
@@ -0,0 +1,13 @@
+<div class="oui-collapsible" data-ng-class="{'oui-collapsible_open':$ctrl.expanded}">
```

Comment:
Unlike in our managers, we don't prefixing with `data-` in our templates

Reply:
The data- prefix has now been removed :)